squirrel.frontier/src/main/java/org/dice_research/squirrel/predictor/RegressionLearn.java

```diff
@@ -0,0 +1,40 @@
+package org.dice_research.squirrel.predictor;
+
+import com.google.common.base.Preconditions;
+import de.jungblut.math.DoubleVector;
+import de.jungblut.math.activation.ActivationFunction;
+import de.jungblut.math.dense.SingleEntryDoubleVector;
+import de.jungblut.math.loss.LossFunction;
+import de.jungblut.math.minimize.CostGradientTuple;
+import de.jungblut.online.minimizer.StochasticMinimizer;
+import de.jungblut.online.ml.FeatureOutcomePair;
+import de.jungblut.online.regression.RegressionLearner;
+
+
+public class RegressionLearn extends RegressionLearner {
```

Comment:
Javadoc comments are missing.

I am wondering whether this class could be merged with the `PredictorBuilder` implementation proposed above :thinking: